scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -219,8 +219,11 @@ def compute_config(args, cpu_flags=cpuflags(), machine=None):
         target_dev=targetdevprefix + 'a',
         bootorder=args.bootorder)
 
-    return get_config(merge_dicts(values, configopts),
-                      os.path.join(TEMPLATE_DIR, "compute-node.xml"))
+    config = get_config(merge_dicts(values, configopts),
+                        os.path.join(TEMPLATE_DIR, "compute-node.xml"))
+    if not hypervisor_has_virtio(libvirt_type):
+        config = re.sub(r"<address type='pci'[^>\n]*>", "", config)
```

Comment:
`[^>\n]` should not contain `\n` as it is correct XML with having newlines within an element tag.